training/run_eval.py

```diff
@@ -649,6 +673,12 @@ def compute_metrics(pred_str, label_str):
         "no_speech_threshold": data_args.no_speech_threshold,
     }
 
+    forced_decoder_ids = processor.get_decoder_prompt_ids(
+        task=data_args.task, 
+        language=data_args.language, 
+        no_timestamps=data_args.return_timestamps
```

Comment:
This is a confusing API - usually we'd work on making the API more consistent (i.e. all timestamp arguments are `return_timestamps`, rather than `no_timestamps` and `decode_timestamps`), but since we've deprecated the `forced_decoder_ids` functionality in Transformers, we can leave it as is.
```suggestion
        no_timestamps=not data_args.return_timestamps
```